js/front/dataset/index.js

```diff
@@ -216,30 +219,18 @@ new Vue({
         checkResource(resource) {
             const url = parseUrl(resource.contentUrl);
             const resource_el = document.querySelector(`#resource-${resource['@id']}`);
-            const el = resource_el.querySelector('.format-label');
+            const el = resource_el.querySelector('.healthcheck-container');
```

Comment:
Resource check logic encapsulation will allow to use `v-el` instead of raw query selector